<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSWindow</class>
 <widget class="QMainWindow" name="QSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scoreur</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>940</width>
      <height>360</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Wave View</string>
     </attribute>
     <widget class="QGraphicsView" name="wavView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>936</width>
        <height>204</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>select zone</string>
      </property>
      <property name="toolTipDuration">
       <number>1000</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="keyView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>204</y>
        <width>936</width>
        <height>130</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>click me</string>
      </property>
      <property name="toolTipDuration">
       <number>1500</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Score View</string>
     </attribute>
     <widget class="QGraphicsView" name="scoreView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>300</height>
       </rect>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>0</y>
        <width>20</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>300</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQS">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_File_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuConvert">
    <property name="title">
     <string>Convert</string>
    </property>
    <addaction name="actionShow_specturm"/>
    <addaction name="separator"/>
    <addaction name="actionWav_to_score"/>
    <addaction name="actionScore_to_wav"/>
    <addaction name="separator"/>
    <addaction name="actionPreset"/>
   </widget>
   <widget class="QMenu" name="menuOpened">
    <property name="title">
     <string>Opened</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuQS"/>
   <addaction name="menuConvert"/>
   <addaction name="menuOpened"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string extracomment="Current Path: "/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout_QS">
   <property name="text">
    <string>About QS</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File ...</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File ...</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWav_to_score">
   <property name="text">
    <string>wav to score</string>
   </property>
  </action>
  <action name="actionScore_to_wav">
   <property name="text">
    <string>score to wav</string>
   </property>
  </action>
  <action name="actionSave_File_as">
   <property name="text">
    <string>Save File as ...</string>
   </property>
  </action>
  <action name="actionPreset">
   <property name="text">
    <string>preset</string>
   </property>
  </action>
  <action name="actionShow_wave_form">
   <property name="text">
    <string>show wave form</string>
   </property>
  </action>
  <action name="actionShow_specturm">
   <property name="text">
    <string>show specturm</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen_File</sender>
   <signal>triggered()</signal>
   <receiver>statusBar</receiver>
   <slot>clearMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
